### PR TITLE
add opm status command

### PR DESCRIPTION
**Pull request by lauraGgit**
This command pings the opm api and returns the current status with a summary and a green light/stoplight/yellow light emoji for what the alert is.

### CODE REVIEW

**Comment by lauraGgit** [comment]
I tested this locally and it seemed to work correctly. I would love to have this up and running before Snowmaddegon II!

**Comment by afeld** [comment]
Love it!

File: scripts/opm_status.coffee [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+#
+# Author:
+#   lauraggit
+
+
+# :greenlight: :redlight: :yellowlight:
+icons = 
+  'Open': ':greenlight:'
+  'Alert': ':yellowlight:'
+  'Closed': ':yellowlight:'
```
Comment (monfresh) [inline scripts/opm_status.coffee]: Shouldn't this be `:redlight:` ?

Comment (lauraGgit) [inline scripts/opm_status.coffee]: Oh yes, probably!

On Fri, Jan 22, 2016 at 10:33 AM, Moncef Belyamani <notifications@github.com

**Comment by OpenGlobe** [comment]
Well done - nice work!

File: scripts/opm_status.coffee [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+module.exports = (robot) ->
+  robot.respond /opm status/i, (msg) ->
+    msg.http("https://www.opm.gov/json/operatingstatus.json").get() (err, res, body) ->
+      if err || res.statusCode != 200
+        msg.send "Well, what does Capital Weather Gang say?"
+      else
+        status = JSON.parse(body)
+        msg.send status['Icon'] + ' ' + icons[status['Icon']] + ' - ' + status['StatusSummary'] + ' - ' + '<' + status['Url']+ '|More>'
```
Comment (monfresh) [inline scripts/opm_status.coffee]: What is the purpose of the `|More` ? What do you think about having `status['Url']` by itself, without the `<` and `|More>`? That way, the link will be clickable in Slack (presumably).

Comment (lauraGgit) [inline scripts/opm_status.coffee]: I was going off of the slack documentation for links, but I can just scrap that and just have the url link you suggested.

**Pull request merged**